user/details/script.js

```diff
@@ -382,6 +393,10 @@ function getMonth(index) {
   return months[index];
 }
 
+function generateReadableDateFromTimeStamp(timeStamp) {
+  return new Date(timeStamp).toDateString();
```

Comment:
If timestamp is invalid this will return `invalid date`, is this expected?

Reply:
Usually this timestamp will be the timestamp what the user has entered from my site. We have a validation check in Backend as well as My Site so that user don't enter a irrelevant data.